tests/spread/plugins/plainbox/provider-basic/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Ensure a basic provider works
+
+systems: [ubuntu-16*, ubuntu-18*]
+
+environment:
+  SNAP_DIR: ../snaps/provider-basic
```

Comment:
I would define an env called SNAPS_DIR  (or similar) at spread.yaml level pointing to ../snaps/ (use absolute path if possible as we go with the tests_lib path https://github.com/snapcore/snapd/blob/master/spread.yaml#L11), and then you can define SNAP_DIR: $SNAPS_DIR/provider-basic

Reply:
I will consider this for a future refactor, as all our snaps live within the suites themselves (this relative path does not leave the suite level)

Reply:
We can do this in a future PR that is more widespread, I don't want to deviate from the current layout in just this PR.